packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -1014,10 +1015,13 @@ class ReorderableDragStartListener extends StatelessWidget {
   /// The index of the associated item that will be dragged in the list.
   final int index;
 
+  /// Whether the associated item can be dragged in the list
+  final bool draggable;
```

Comment:
I wonder if `enabled` might be a better name here as this effectively shuts off the listener altogether.

Reply:
Yes, I also hesitated with this name. Let's call it `enabled` then